Mathlib/Topology/DiscreteSubset.lean

```diff
@@ -286,9 +286,10 @@ theorem discreteTopology_biUnion_finset {ι : Type*} {I : Finset ι} {s : ι →
   simpa [biInter_finset_mem I] using fun i hi ↦ compl_mem_codiscrete_iff.mpr ⟨hs' i hi, hs i hi⟩
 
 /-- The union of finitely many discrete closed subsets is discrete. -/
-theorem discreteTopology_iUnion_fintype {ι : Type*} [Fintype ι] {s : ι → Set X}
+theorem discreteTopology_iUnion_fintype {ι : Type*} [Finite ι] {s : ι → Set X}
```

Comment:
- Should we rename it to `*_iUnion_finite` or even `*.iUnion_finite`?
- Should we use the recently introduced `IsDiscrete` predicate here?
- Offtopic: should `IsDiscrete` be namespaced?